starter_templates/redis/rust/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "rustredis"
+version = "0.1.0"
+authors = ["Syed Ghufran Hassan <syedghufranhassan@gmail.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[[bin]]
+name = "main"
+path = "/app/main.rs"
+[dependencies]
```

Comment:
If there are no dependencies, do we need this `[dependencies]` block?

Reply:
Sir actually I did it because my docker file was not running without mentioning them in dependencies.